src/docs/config/security.config.ts

```
@@ -1,19 +1,28 @@
-import { ThrottlerModuleOptions } from '@nestjs/throttler';
 import { ConfigService } from '@nestjs/config';
-
-export const getThrottlerConfig = (configService: ConfigService): ThrottlerModuleOptions => {
-  return {
-    ttl: configService.get<number>('THROTTLE_TTL') || 900, // 15 minutes in seconds
-    limit: configService.get<number>('THROTTLE_LIMIT') || 100,
-  };
+import { ThrottlerModule } from '@nestjs/throttler';
+import * as crypto from 'crypto';
```

ESLint

[error] 'crypto' is defined but never used.

(@typescript-eslint/no-unused-vars)